src/Menu/Dashboard.php

```diff
@@ -70,12 +100,39 @@ protected function register_navigation_pages(): void {
 				'id'       => 'google-dashboard',
 				'title'    => __( 'Dashboard', 'google-listings-and-ads' ),
 				'parent'   => 'google-listings-and-ads-category',
-				'path'     => '/google/dashboard',
+				'path'     => self::$path,
 				'nav_args' => [
 					'order'  => 10,
 					'parent' => 'google-listings-and-ads-category',
 				],
 			]
 		);
 	}
+
+	/**
+	 * Maybe redirect to start page.
+	 *
+	 * @return void
+	 */
+	public function maybe_redirect_to_start_page(): void {
+		if ( $this->wp->wp_doing_ajax() ) {
+			return;
+		}
+
+		if ( ! $this->is_current_wc_admin_page( self::$path ) ) {
+			return;
+		}
+
+		$this->redirect_to_start_page();
+	}
+
+	/**
+	 * Redirect to start page and exit.
+	 *
+	 * @return void
+	 */
+	private function redirect_to_start_page(): void {
+		wp_safe_redirect( admin_url( 'admin.php?page=wc-admin&path=' . rawurlencode( GetStarted::$path ) ) );
+		exit;
```

Comment:
See https://github.com/woocommerce/google-listings-and-ads/pull/1926/files#r1143240315